WatchDogCore/WatchDog/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,12 @@
+eclipse.preferences.version=1
```

Comment:
Can you some research into whether they are really needed (I don't think so). If not, remove them everywhere please